blog/2024/05-15-backing-up-jellyfin/index.mdx

```diff
@@ -0,0 +1,67 @@
+
+Finally, backups are critical if you want to test Unstable versions, for all the reasons above. In fact, when testing Unstable, it is best to back up much more regularly, perhaps using a cron job or similar automatic mechanism, to ensure that your data can be recovered if you hit a major issue.
+
+Hopefully this outlines why backups of your Jellyfin data are important. Next, we'll cover how to take backups and restore from them should you need to.
+
+## Taking a Backup
+
+Here is the process to take a Jellyfin backup, for every platform we support.
+
+1. Stop the running Jellyfin server process. This is extremely important, as otherwise the database will be locked and might not be recoverable when restoring.
+
+   * Any platform: Within the Jellyfin Dashboard, click "Shutdown". This should cleanly stop the process on all platforms, but if not, try one of the following.
+   * Docker: `sudo docker stop jellyfin`
+   * Debian/Ubuntu packages: `sudo systemctl stop jellyfin` or `sudo service jellyfin stop`
+   * Windows Installer: Right-click the Tray app, and select "Quit"; or, in Process Manager, find the Jellyfin process and terminate it.
+   * MacOS Installer (.dmg): In Activity Monitor, find the Jellyfin process and terminate it.
+   * Portable Installs (regardless of platform): Stop the running `jellyfin` or `jellyfin.exe` program, however it was started.
+
+2. Copy your data and configuration directories. What you name the copies is up to you; personally, I like to use dated and versioned directory names e.g. `jellyfin.2024-05-01_10.8.13`. Where these files are also depends on the platform. I provide terminal commands for Docker, Debian/Ubuntu, and MacOS but you can also perform these tasks from a graphical file browser via copy + paste + rename.
+
+  * Docker: Wherever your `/data` and `/config` volumes are sourced from. Assuming they're stored in somewhere like `/srv/jellyfin`, execute `sudo cp -a /srv/jellyfin/data /srv/jellyfin/data.2024-05-01_10.8.13` and `sudo cp -a /srv/jellyfin/config /srv/jellyfin/config.2024-05-01_10.8.13`.
+  * Debian/Ubuntu packages: Data is in `/var/lib/jellyfin` and config is in `/etc/jellyfin` by default. Execute `sudo cp -a /var/lib/jellyfin /var/lib/jellyfin.2024-05-01_10.8.13` and `sudo cp -a /etc/jellyfin /etc/jellyfin.2024-05-01_10.8.13`.
+  * Windows Installer:
+  * MacOS Installer (.dmg): Data is stored in one of these paths; back up whichever one(s) exist: `~/.config/jellyfin/`, `~/.local/share/jellyfin/`, `~/Library/Application Support/Jellyfin/`.
+  * Portable Installs:
+     * Linux:
+     * Windows:
+     * MacOS: Data is stored in one of these paths; back up whichever one(s) exist: `~/.config/jellyfin/`, `~/.local/share/jellyfin/`, `~/Library/Application Support/Jellyfin/`.
+
```

Comment:
If the user is really unsure, the data dir will be printed in the log during startup